.github/workflows/pypi_publish.yml

```diff
@@ -48,8 +59,122 @@ jobs:
           name: wheels-sdist
           path: ./wheelhouse/*.tar.gz
 
+  build-wheels:
+    name: 'Wheels: ${{ matrix.os }} ${{ matrix.arch }} ${{ matrix.python-version }}'
+    runs-on: ${{ matrix.os }}
+    continue-on-error: true
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        python-version: ['3.9', '3.10', '3.11', '3.12', '3.13']
+        include:
+          - os: ubuntu-latest
+            arch: x86_64
+          - os: windows-latest
+            arch: AMD64
+          - os: macos-latest
+            arch: arm64
+
+    steps:
+    - name: Check out the repo
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0  # get the non-merge commit for PRs
+        fetch-tags: true # include tags to get correct version from setuptools_scm
+
+    # Don't use conda to build pip wheels
+    # (it's going to be messy when you import the packages outside of conda)
+    - name: 🐍 Set up Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    # MacOS environment setup
+    # libz and libhdf5 require MacOS>=14.0
+    - name: Install openmpi; update env variables on macos
```

Comment:
I'm surprised that h5py (which does explicitly use mpi if installed) doesn't need this section, whereas versioned-hdf5 (which doesn't have any mpi-specific code) needs it?

Reply:
Yep, I'm fine if we remove this hard requirement. IMO it should be optional, and would greatly decrease the complexity of the packaging.